akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1196,16 +1180,15 @@ trait FlowOps[+Out, +Mat] {
    *        that are supported; if more distinct keys are encountered then the stream fails
    */
   def groupBy[K](maxSubstreams: Int, f: Out ⇒ K): SubFlow[Out, Mat, Repr, Closed] = {
```

Comment:
Is `@param f` scaladoc omitted on purpose? Should it be clarified here what happens when `f` returns `null`? This should then be enforced by the impl.